scripts/generate/templates/Checkpoints.js

```diff
@@ -163,6 +189,12 @@ function _upperBinaryLookup(
     return high;
 }
 
+/**
+ * @dev Return the index of the first checkpoint for which \`key\` <= checkpoint.key, i.e. the oldest checkpoint which key is higher or equal than the search key.
```

Comment:
The first part is confusing due to ambiguity of the first occurence of the word "key".

```suggestion
 * @dev Return the index of the oldest checkpoint whose key is greater or equal than the search key, or `high` if there is none.
```